### ISSUE TITLE
Can't view currently blacklisted members in context of discord

### ISSUE THREAD

**Issue by ItsAzaria** [the-programmers-hangout/aura#2]
Needs a command just to view all blacklisted members

**Comment by ghost** [comment]
Inside of BlockService, something of the sort should make the deal:
```
    def find_blacklisted_all(self, guild_id: str):
        return self._blacklist.find(filter=dict(guild_id=guild_id)))
```
Iterate over the cursor in a command and put everything into a \n seperated string that is returned by aura. Or if we fear that the blacklist will get too many entries -> output an embedded text file?

**Comment by ItsAzaria** [comment]
Few options
 - split it over a few messages
 - paginated embed
 - text file

**Comment by ghost** [comment]
implemented in 1.5.0
